packages/cysync-core-constants/src/i18n/lang/ar-AE.json

```
@@ -2152,7 +2193,9 @@
           "secondaryEmailField": { "label": "Secondary Email" },
           "reminderPeriodField": {
             "label": "Reminder Perioc",
-            "input": "Every ${month} month"
+            "input": "Every ${month} ",
+            "month": "month",
+            "months": "months"
```

This will work in other languages as well.

```suggestion
            "input": "Every ${month} month ",
            "inputPlural": "Every ${month} months",
```